tests/ui/generic-associated-types/issue-87258_a.stderr

```diff
@@ -1,10 +1,15 @@
-error: unconstrained opaque type
+error: item does not constrain `<&'c mut S as Trait2>::FooFuture::{opaque#0}`
+  --> $DIR/issue-87258_a.rs:18:8
+   |
+LL |     fn foo<'a>() -> Self::FooFuture<'a> {
+   |        ^^^
+   |
+   = note: consider removing `#[define_opaque]` or adding an empty `#[define_opaque()]`
```

Comment:
This note will need tweaking

Reply:
Yeah, that's existing though and the suggestiono to add an empty `#[define_opaque()]` is applicable for ATPIT afaik?

Reply:
Yes, the "consider removing" part should be dependent on the existence of the attr, tho.